internal/clients/new_instance_types_test.go

```diff
@@ -0,0 +1,74 @@
+package clients_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/RHEnVision/provisioning-backend/internal/clients"
+	"github.com/RHEnVision/provisioning-backend/internal/clients/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+
+	"github.com/aws/smithy-go/ptr"
+	"github.com/stretchr/testify/assert"
+)
```

Comment:
Was going to recommend importing `_ "github.com/RHEnVision/provisioning-backend/internal/testing/initialization"` but then I realized you only test these functions, nice.